### DISCUSSION TITLE
Novel audio storage implementation

### DISCUSSION THREAD

**Discussion by EvESpirit** [mrusse/KCS-Utility#10] [General]
Hi!

I've noticed you said you were looking into creating a KCS successor. I've been looking into this topic intensely for several weeks now and I would like to ask if you're interested in collaboration.
If so, let me know here and we can swap the rest in DMs.
If you would like to see my code, I have a repository here called Tracer with an unfinished proof of concept.

**Comment by mrusse** [comment]
> Hi!
> 
> I've noticed you said you were looking into creating a KCS successor. I've been looking into this topic intensely for several weeks now and I would like to ask if you're interested in collaboration. If so, let me know here and we can swap the rest in DMs. If you would like to see my code, I have a repository here called Tracer with an unfinished proof of concept.

Hello! Thanks for the interest but I'm honestly way too busy with work these days to commit to any collaboration. I am definitely interested though and would love to be updated if you make any progress. Let me know if you have any specific questions though I'd be happy to try and help if I can.

**Comment by jnguyen1098** [comment]
Interested in updates too

**Comment by EvESpirit** [comment]
Well, to be honest, collaborating would mean I'm a lot more motivated to push commits out the door, hence why I was asking. I'll hopefully get around to it myself one day but I find that unlikely unless I find someone else lol

I appreciate your reply though. I can understand not having time. Best of luck with whatever you're up to.
